Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -26,7 +26,7 @@ public struct Event: Equatable {
         }
     }
 
-    public struct Identifiers: Equatable {
+    struct Identifiers: Equatable {
```

Comment:
These should have been deprecated...

Reply:
I suppose you could still release a 2.6 where you deprecate it. Kinda pedantic.

Reply:
That was my thought too. I don't think you would use one without the other.